src/NuGet.Core/NuGet.Configuration/PublicAPI.Unshipped.txt

```diff
@@ -1,2 +1,9 @@
 NuGet.Configuration.PackageSourceMapping.PackageSourceMapping(System.Collections.Generic.IReadOnlyDictionary<string, System.Collections.Generic.IReadOnlyList<string>> patterns) -> void
-NuGet.Configuration.NuGetConfigurationException.NuGetConfigurationException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) -> void
\ No newline at end of file
+NuGet.Configuration.NuGetConfigurationException.NuGetConfigurationException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) -> void
+NuGet.Configuration.PackageSourceMapping.Patterns.get -> System.Collections.Generic.IReadOnlyDictionary<string, System.Collections.Generic.IReadOnlyList<string>>
+NuGet.Configuration.PackageSourceMappingProvider
+NuGet.Configuration.PackageSourceMappingProvider.AddOrUpdatePackageSourceMappingSourceItem(NuGet.Configuration.PackageSourceMappingSourceItem packageSourceMappingSourceItem) -> void
```

Comment:
Originally when the package source mapping provider was written, a few different apis were added about addupdate and remove source mapping items. 

Now we're adding a `Save` which is really a combination of the 2. 

Do we need the save API? 

At the minimum, I don't think we need both.

Reply:
I added a save method because I was copying the format of PackageSourceProvider.cs, and it has a save method. I could do without the save method, but it makes it a lot easier for me later to only have to call one method when I am saving changes to the settings made in the UI.